config/scootconfig/workers_modules.go

```diff
@@ -70,13 +61,13 @@ type WorkersLocalConfig struct {
 }
 
 func (c *WorkersLocalConfig) Install(bag *ice.MagicBag) {
-	bag.Put(func(tmp *temp.TempDir) worker.WorkerFactory {
+	bag.Put(func(tmp *temp.TempDir) func(cluster.Node) runner.Service {
 		return InmemoryWorkerFactory(tmp)
```

Comment:
Just saying we have half worker and half service terminology. It's not confusing enough to change anything though.